renovate-changesets/action.yaml

```diff
@@ -1,10 +1,14 @@
-name: Backstage Renovate Changeset Creator
-description: Create changesets on the renovate bot PR's if needed
+name: Backstage Dependency Manager Changeset Creator
```

Comment:
The action is still named `renovate-changesets` though.

Hm. Should we make a new parallel one that has this new ability (and is named `dependency-manager-changesets`)? Possibly deprecating the existing one later down the line in a separate PR.

Alternatively: Should we instead make a new parallel one _specifically_ named `dependabot-changesets` and leave the old one intact?

Reply:
The former would require some upstream changes.
The latter would probably create more duplicated code (or the need to share code)